docker/docker-compose.yaml

```
@@ -55,66 +58,127 @@ services:
+      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID}
     networks:
       - autoware-net
     depends_on:
       - map
     command: >
       ros2 launch tier4_vehicle_launch vehicle.launch.xml
-      launch_vehicle_interface:=${LAUNCH_VEHICLE_INTERFACE:-true}
+      launch_vehicle_interface:=${LAUNCH_VEHICLE_INTERFACE}
       raw_vehicle_cmd_converter_param_path:=/opt/autoware/share/autoware_launch/config/vehicle/raw_vehicle_cmd_converter/raw_vehicle_cmd_converter.param.yaml
-      sensor_model:=${SENSOR_MODEL:-sample_sensor_kit}
-      use_sim_time:=${USE_SIM_TIME:-true}
-      vehicle_id:=${VEHICLE_ID:-default}
-      vehicle_model:=${VEHICLE_MODEL:-sample_vehicle}
-    restart: unless-stopped
+      sensor_model:=${SENSOR_MODEL}
+      use_sim_time:=${USE_SIM_TIME}
+      vehicle_id:=${VEHICLE_ID}
+      vehicle_model:=${VEHICLE_MODEL}
+    restart: on-failure
 
   system:
     image: ghcr.io/autowarefoundation/autoware:universe-vehicle-system
     container_name: autoware-system
     pid: service:map
+    ipc: host
     init: true
     environment:
-      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID:-0}
+      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID}
     networks:
       - autoware-net
     depends_on:
       - map
     command: >
       ros2 launch autoware_launch tier4_system_component.launch.xml
-      launch_system_monitor:=${LAUNCH_SYSTEM_MONITOR:-false}
-      launch_dummy_diag_publisher:=${LAUNCH_DUMMY_DIAG_PUBLISHER:-true}
-      sensor_model:=${SENSOR_MODEL:-sample_sensor_kit}
-      system_run_mode:=${SYSTEM_RUN_MODE:-planning_simulation}
-      use_sim_time:=${USE_SIM_TIME:-true}
-      vehicle_model:=${VEHICLE_MODEL:-sample_vehicle}
-    restart: unless-stopped
+      launch_system_monitor:=${LAUNCH_SYSTEM_MONITOR}
+      launch_dummy_diag_publisher:=${LAUNCH_DUMMY_DIAG_PUBLISHER}
+      sensor_model:=${SENSOR_MODEL}
+      system_run_mode:=${SYSTEM_RUN_MODE}
+      use_sim_time:=${USE_SIM_TIME}
+      vehicle_model:=${VEHICLE_MODEL}
+    restart: on-failure
+
+  sensing-perception:
+    image: ghcr.io/autowarefoundation/autoware:universe-sensing-perception
+    container_name: autoware-sensing-perception
+    pid: service:map
+    ipc: host
+    init: true
+    environment:
+      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID}
+    networks:
+      - autoware-net
+    depends_on:
+      - map
+      - system
+    command: >
+      bash -c "
+      ros2 launch autoware_launch tier4_sensing_component.launch.xml
+      launch_pointcloud_container:=${LAUNCH_POINTCLOUD_CONTAINER_SENSING}
+      launch_sensing_driver:=${LAUNCH_SENSING_DRIVER}
+      pointcloud_container_name:=${POINTCLOUD_CONTAINER_NAME}
+      sensor_model:=${SENSOR_MODEL}
+      use_sim_time:=${USE_SIM_TIME}
+      vehicle_model:=${VEHICLE_MODEL} &
+      ros2 launch autoware_launch tier4_perception_component.launch.xml
+      data_path:=${DATA_PATH}
+      launch_pointcloud_container:=${LAUNCH_POINTCLOUD_CONTAINER_PERCEPTION}
+      lidar_detection_model:=${LIDAR_DETECTION_MODEL}
```

We should if we want to run the perception without GPU, we should select "laserscan_based" method for occupancy grid map generation.
```suggestion
      lidar_detection_model:=${LIDAR_DETECTION_MODEL}
      occupancy_grid_map_method:=${OCCUPANCY_GRID_MAP_METHOD}
```